app/threads/client/lib/normalizeThreadTitle.js

```diff
@@ -26,6 +27,9 @@ export const normalizeThreadTitle = ({ ...message }) => {
 			roomTemplate: ({ prefix, mention }) => `${ prefix }<strong> ${ mention } </strong>`,
 		});
 		const { html } = emojiParser({ html: filteredMessage });
+		if (message.title) {
```

Comment:
I guess it would be cleaner to do something like
```
const parsedInstace = message.title ? instance.parse() : instance.parse()
return parsedInstance
```

Reply:
Corrected, thanks